packages/article-skeleton/src/article-body/inline-paragraph.js

```diff
@@ -52,7 +52,9 @@ const InlineParagraph = ({
   const positioned = manager.layout();
 
   return [
-    dropCap && dropCap.element,
+    dropCap && (
+      <View style={{ left: gutters - spacing(1) }}>{dropCap.element}</View>
```

Comment:
```suggestion
      <View style={{ left: gutters - spacing(2) }}>{dropCap.element}</View>
```
Do you think we have to subtract 10px (spacing(2)) in order to keep the current alignment from production:

![image](https://user-images.githubusercontent.com/8720661/85519498-46160d80-b60a-11ea-9f20-a1045f54b4c8.png)


![image](https://user-images.githubusercontent.com/8720661/85519629-765dac00-b60a-11ea-9eca-002e100b3d5e.png)

Reply:
Yeah good spot @infreezer - - just tested this on an iPad and definitely looks more aligned. I'll update to `spacing(2)`